ci-operator/step-registry/osd/create/create/osd-create-create-commands.sh

```diff
@@ -79,18 +80,25 @@ fi
 
 CLUSTER_INFO="${ARTIFACT_DIR}/ocm-cluster.txt"
 
+CLUSTER_MULTI_AZ_SWITCH=""
+if $CLUSTER_MULTI_AZ; then
+  CLUSTER_MULTI_AZ_SWITCH="--multi-az"
+fi
```

Comment:
You should probably ensure that the value of `$CLUSTER_MULTI_AZ` is actually what you expect, instead of relying on testing the return code of a shell builtin.  Something like:
``` bash
if [[ "$CLUSTER_MULTI_AZ" == "true" ]]; then
  CLUSTER_MULTI_AZ_SWITCH="--multi-az"
fi
```

Reply:
done